### DISCUSSION TITLE
Chewing Tobacco and Nicotine Gum have no effect other than mild stimulation (Win)

### DISCUSSION THREAD

**Discussion by RodriguesCIA** [cataclysmbn/Cataclysm-BN#2065] [General]
One would expect these to be a good source of the Nicotine buff without having to lit yourself up or with a lesser risk for addiction, but not quite. Mostly pointless, to say the least.

**Comment by Coolthulhu** [comment]
It's supposed to be only really useful to stopping the addiction debuff. But using them for that is rather tedious, because addiction mechanics are quite old and not well done.
Buffing them by allowing nicotine effect on them would make them overtake electronic cigs, which are just nicotine+(weak) morale.

I'm not sure what would be the best option here.
